README.md

```diff
@@ -1,87 +1,105 @@
 # dcos-metrics
-[![Build Status](https://jenkins.mesosphere.com/service/jenkins/buildStatus/icon?job=public-dcos-metrics/public-dcos-metrics-master)](https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-metrics/job/public-dcos-metrics-master/)
-[![Go Report Card](https://goreportcard.com/badge/github.com/dcos/dcos-metrics)](https://goreportcard.com/report/github.com/dcos/dcos-metrics)
+[![Build Status](https://jenkins.mesosphere.com/service/jenkins/buildStatus/icon?job=public-dcos-metrics/public-dcos-metrics-master)][jenkins-master]
+[![Go Report Card](https://goreportcard.com/badge/github.com/dcos/dcos-metrics)][go-report-card]
 
-_*Note:* This project is a work-in-progress. We're currently aiming to ship a completed service,
-with integration hooks, as part of DC/OS 1.10. Community help is welcome and appreciated!_
-
-1. [Overview](#overview)
-2. [How this repo is organized](#how-this-repo-is-organized)
-3. [Getting Started](#getting-started)
-4. [Documentation](#documentation)
-5. [Community](#community)
-6. [Contributing](#contributing)
-7. [License](#license)
-8. [Acknowledgements](#acknowledgements)
+**Table of Contents:**
+  1. [Overview](#overview)
+  2. [How This Repo Is Organized](#how-this-repo-is-organized)
+  3. [Getting Started](#getting-started)
+  4. [Documentation](#documentation)
+  5. [Community](#community)
+  6. [Contributing](#contributing)
+  7. [License](#license)
+  8. [Acknowledgements](#acknowledgements)
 
 ## Overview
+As a user of DC/OS, you want to be able to retrieve discrete metrics about your applications and cluster(s) in order to
```

Comment:
Suggest: `To gain operational insight as a user of DC/OS, you need the ability to retrieve discrete metrics about your applications and clusters.`